github/github_test.go

```diff
@@ -3117,9 +3117,21 @@ func TestDeploymentProtectionRuleEvent_GetRunID(t *testing.T) {
 		t.Errorf("want %#v, got %#v", want, got)
 	}
 
-	want = -1
-	url = "repos/dummy-org/dummy-repo/actions/runs/abc123/deployment_protection_rule"
+	want = 123456789
+	url = "https://local.dummyhost.net/repos/dummy-org/dummy-repo/actions/runs/123456789/deployment_protection_rule"
```

Comment:
Actually, on GitHub Enterprise, the URL comes back in the event without the base URL, like the following, so please change this line to represent that.

```suggestion
	// GitHub Enterprise returns the URL in events without the leading domain:
	url = "repos/dummy-org/dummy-repo/actions/runs/123456789/deployment_protection_rule"
```